### PR TITLE
sshguard: minor tweaks

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
what does "whinging" mean?

File: Formula/sshguard.rb [hunk 1]
```diff
@@ -21,11 +21,11 @@ def install
     system "make", "install"
     cp "examples/sshguard.conf.sample", "examples/sshguard.conf"
     inreplace "examples/sshguard.conf" do |s|
-      s.gsub! /^#BACKEND=.*$/, "BACKEND=\"#{libexec}/sshg-fw-#{firewall}\""
```
Comment (ilovezfs) [inline Formula/sshguard.rb]: I'm not getting any audit error for this.

**Comment by DomT4** [comment]
I fingered the wrong tool. It's `ruby` complaining rather than `rubocop`, with the standard whinge here:

```
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/sshguard.rb:24: warning: ambiguous first argument; put parentheses or a space even after `/' operator
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/sshguard.rb:28: warning: ambiguous first argument; put parentheses or a space even after `/' operator
Syntax OK
```

**Comment by ilovezfs** [comment]
:tangerine:

**Comment by DomT4** [comment]
Ta ❤️

**Pull request closed without merging**